README.md

```diff
@@ -140,35 +139,51 @@ pip install .
 ```
 
 On the master node, with a unique node name of $NODE_NAME, for a cluster wide
-private address range of $CLUSTER_IP_SUBNET and a smaller
-subnet for the logical switch created in the master node of
-$MASTER_SWITCH_SUBNET, run:
+private address range of $CLUSTER_IP_SUBNET and a smaller subnet for the
+logical switch created in the master node of $MASTER_SWITCH_SUBNET, run:
 
 ```
-ovn-k8s-overlay master-init --cluster-ip-subnet=$CLUSTER_IP_SUBNET \
- --master-switch-subnet="$MASTER_SWITCH_SUBNET" --node-name="$NODE_NAME"
+ovn-k8s-overlay master-init \
+  --cluster-ip-subnet=$CLUSTER_IP_SUBNET \
+  --master-switch-subnet="$MASTER_SWITCH_SUBNET" \
+  --node-name="$NODE_NAME"
 ```
 
 An example is:
 
 ```
-ovn-k8s-overlay master-init --cluster-ip-subnet="192.168.0.0/16" \
---master-switch-subnet="192.168.1.0/24" --node-name="kube-master"
+ovn-k8s-overlay master-init \
+  --cluster-ip-subnet="192.168.0.0/16" \
+  --master-switch-subnet="192.168.1.0/24" \
+  --node-name="kube-master"
 ```
 
 The above command will create a cluster wide logical router, a connected
 logical switch for the master node and a logical port and a OVS internal
 interface named "k8s-$NODE_NAME" with an IP address via which other nodes
-should be eventually able to reach the daemons running on this node.
-This IP address will be referred in the future as $OVN_CENTRAL_IP.
+should be eventually able to reach the daemons running on this node. This IP
+address will be referred in the future as $K8S_API_SERVER_IP.
 
 * k8s minion node initializations.
 
 Set the k8s API server address in the Open vSwitch database for the
-initialization scripts (and later daemons) to pick from.
+initialization scripts (and later daemons) to pick from. For insecure
+connections:
+
+```
+ovs-vsctl set Open_vSwitch . \
+  external_ids:k8s-api-server="$K8S_API_SERVER_IP:8080"
+```
+
+For secure connections $API_TOKEN should be provided. In case of self-signed
+certificates $CA_CRT should be present in /etc/openvswitch/k8s-ca.crt or
+stored in the OVSDB:
```

Comment:
I would consider adding somewhere something like the following:

please note that: if no protocol is specified in external_ids:k8s-api-server, insecure http connection will be used.